### PR TITLE
Completed css-pseudo-classes

### PR DESCRIPTION
**Pull request by julieluangpraxay**
Screenshot 2023-07-19 at 2 38 58 PM

### CODE REVIEW

File: css-pseudo-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      <tr>
+        <td>Kent Beck</td>
+        <td>Cascading Style Sheets</td>
+        <td>90</td>
+      </tr>
+      <tr>
+        <td>Kyle Simpson</td>
+        <td>JavaScript</td>
+        <td>99</td>
+      </tr>
```
Comment (shawnkost) [inline css-pseudo-classes/index.html:47]: This should be inside a `tbody`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here!

File: css-pseudo-classes/index.html (hunk 1 above)
Comment (julieluangpraxay) [reply css-pseudo-classes/index.html:47]: Screenshot 2023-07-22 at 10 51 01 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**